### PR TITLE
add index on block number to address token balances

### PR DESCRIPTION
**Pull request by ayrat555**
*[GitHub keywords to close any associated issues](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)*

## Motivation

I think the problem with importing new blocks is not in connection pool but  is that import query can not be finished because some sub queries take too long to execute. I see in pgadmin some requests that are still executing from my tests. 

https://github.com/poanetwork/blockscout/pull/1323

## Changelog

- add index on block number to address token balances

### CODE REVIEW

File: apps/explorer/mix.exs [hunk 1]
```diff
@@ -91,7 +91,7 @@ defmodule Explorer.Mixfile do
       {:math, "~> 0.3.0"},
       {:mock, "~> 0.3.0", only: [:test], runtime: false},
       {:mox, "~> 0.4", only: [:test]},
-      {:poison, "~> 3.1", only: [:test]},
+      {:poison, "~> 3.1"},
```
Comment (zachdaniel) [inline apps/explorer/mix.exs]: why this change?

Comment (ayrat555) [reply apps/explorer/mix.exs]: migration generation fails with `poison.app` is not found
